wordpress-mobile/WordPress-iOS
# Notifications: Badge Optimization

#### Details:
This PR cleans up the Notifications Badge's Image helpers. A single download will be fired, per URL.

#### To test:
1. Log into a dotcom account
2. Open a Badge Notification

Please, verify that the image shows up with a single spring animation (instead of multiple, which is what was happening in develop!).

Needs review: @diegoreymendez 
Thanks in advance!
